pkg/tests/end_to_end_tests/jaeger_store_integration_test.go

```diff
@@ -1,3 +1,5 @@
+//go:build jaeger_storage_test
```

Comment:
It does seem that we will run Jaeger tests two times. (1) when we run e2e tests (2) when we run only Jaeger storage tests with verbose?